client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -473,8 +376,106 @@ def get_attribute_defs(cls):
                     "Visibility state will be stored in the Alembic file. "
                     "Otherwise everything written out is treated as visible."
                 )
+            ),
+            "preRollStartFrame": NumberDef(
+                "preRollStartFrame",
+                label="Pre Roll Start Frame",
+                tooltip=(
+                    "The frame to start scene evaluation at. This is used"
+                    " to set the starting frame for time dependent "
+                    "translations and can be used to evaluate run-up that"
+                    " isn't actually translated."
+                ),
+                default=cls.preRollStartFrame
+            ),
```

Comment:
Can we put `Pre Roll` toggle in front of this one? They are closely related? (I know they were together before this last commit - just anticipated that if they were moved that they would be moved together.)

Reply:
Done